### PR TITLE
 Reuse microos/journal_check in JeOS testing 2nd attempt

### PR DESCRIPTION
**Pull request by mloviska**
- Related ticket: [JeOS's Warnings in log](https://progress.opensuse.org/issues/93267)
- Verification runs: 
   * [microos-Tumbleweed-MicroOS-Image](https://openqa.opensuse.org/tests/1831072#details)
   * [microos-Tumbleweed-MicroOS-Image-Kubic-kubeadm](https://openqa.opensuse.org/tests/1831071#live)
   * [microos-Tumbleweed-MicroOS-Image-ContainerHost](https://openqa.opensuse.org/tests/1831070#live)
   * [sle-15-SP3-JeOS-for-kvm-and-xen-x86_64-Build2.36-log-warnings@svirt-xen-pv](http://kepler.suse.cz/tests/5879)
   * [sle-15-SP3-JeOS-for-VMware-x86_64-Build2.36-log-warnings@svirt-vmware65](http://kepler.suse.cz/tests/5877#step/journal_check/1)
   * [sle-15-SP3-JeOS-for-kvm-and-xen-x86_64-Build2.36-log-warnings@svirt-xen-hvm](http://kepler.suse.cz/tests/5878#step/journal_check/1)
   * [sle-15-SP3-JeOS-for-kvm-and-xen-x86_64-Build2.36-log-warnings@64bit-virtio-vga](http://kepler.suse.cz/tests/5880#step/journal_check/1)
   * [opensuse-Tumbleweed-JeOS-for-kvm-and-xen-x86_64-Build20210706-log-warnings@uefi_virtio-2G](http://kepler.suse.cz/tests/5882#step/journal_check/1)
   * [sle-15-SP3-JeOS-for-MS-HyperV-x86_64-Build2.36-log-warnings@svirt-hyperv-uefi](http://kepler.suse.cz/tests/5881#step/journal_check/1)
   * [opensuse-15.2-JeOS-for-kvm-and-xen-x86_64-Build31.499-log-warnings@uefi_virtio-2G](http://kepler.suse.cz/tests/5886#live)
   * [opensuse-15.3-JeOS-for-kvm-and-xen-x86_64](https://openqa.opensuse.org/tests/1831081)
   * [sle-15-SP3-JeOS-for-kvm-and-xen-aarch64](https://openqa.suse.de/tests/6397231)

### CODE REVIEW

**Comment by Vogtinator** [comment]
The commits aren't just plain reverts, so the subject+description need to be edited IMO.

The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...

**Comment by mloviska** [comment]
> The commits aren't just plain reverts, so the subject+description need to be edited IMO.
> 
> The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...

Good point, I have changed only the first commit message, as the `microos/libzypp_config.pm` is basically untouched. Let me check the TW failure.

**Comment by mloviska** [comment]
> The commits aren't just plain reverts, so the subject+description need to be edited IMO.
> 
> The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...

Seems like it is somehow triggered by jeos-firstboot. The message itself does not appear after reboot.

**Comment by Vogtinator** [comment]
> > The commits aren't just plain reverts, so the subject+description need to be edited IMO.
> > The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...
> 
> Seems like it is somehow triggered by jeos-firstboot. The message itself does not appear after reboot.

Indeed, `jeos-firstboot` also attempts DHCP on `lo`. I'll send a PR, that can be used as bugref as well.

**Comment by Vogtinator** [comment]
> > > The commits aren't just plain reverts, so the subject+description need to be edited IMO.
> > > The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...
> > 
> > 
> > Seems like it is somehow triggered by jeos-firstboot. The message itself does not appear after reboot.
> 
> Indeed, `jeos-firstboot` also attempts DHCP on `lo`. I'll send a PR, that can be used as bugref as well.

https://github.com/openSUSE/jeos-firstboot/pull/83

**Comment by mloviska** [comment]
> > > > The commits aren't just plain reverts, so the subject+description need to be edited IMO.
> > > > The TW JeOS test now fails with `Jul 08 07:42:37.643071 localhost wicked[755]: DHCPv4 not supported on 'lo'`, never seen that before...
> > > 
> > > 
> > > Seems like it is somehow triggered by jeos-firstboot. The message itself does not appear after reboot.
> > 
> > 
> > Indeed, `jeos-firstboot` also attempts DHCP on `lo`. I'll send a PR, that can be used as bugref as well.
> 
> [openSUSE/jeos-firstboot#83](https://github.com/openSUSE/jeos-firstboot/pull/83)

Thanks, VR:
[opensuse-Tumbleweed-JeOS-for-kvm-and-xen-x86_64-Build20210706-log-warnings@uefi_virtio-2G](http://kepler.suse.cz/tests/5896#)

File: data/journal_check/bug_refs.json [hunk 1]
```diff
@@ -0,0 +1,198 @@
+{
+    "jeos-firstboot_PR#83": {
```
Comment (Vogtinator) [inline data/journal_check/bug_refs.json:2]:
```suggestion
    "gh#openSUSE/jeos-firstboot#83": {
```

**Review by Vogtinator — APPROVED** [review]

**Comment by jlausuch** [comment]
Do we want to keep journal check before all console tests?

**Comment by mloviska** [comment]
> Do we want to keep journal check before all console tests?

I am aware that it is not really following the requirements set by Vincent to catch even package messages as came from the salt etc. But as of now, with log level priority defined to `warning` it feels kinda overwhelming already. It would be nice to have an easier start :zany_face:

**Review by jlausuch — APPROVED** [review]
LGTM

**Pull request merged**